easybuild/easyconfigs/r/r-inla/r-inla-21.01.18-foss-2020b.eb

```diff
@@ -0,0 +1,77 @@
+easyblock = 'MakeCp'
+
+name = 'r-inla'
+version = '21.01.18'
+
+homepage = 'https://www.r-inla.org/'
+description = """R-INLA is a package in R that do approximate Bayesian inference for Latent Gaussian Models."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://hrue.r-inla-download.org/']
+sources = [{
+    'download_filename': 'build64.tgz',
+    'filename': SOURCE_TGZ,
+}]
+patches = ['r-inla-21.01.18-foss2020a-skip_binaries.patch']
+checksums = [
+    '5cf0646f0c6f901b6fd1c5c60345713e02a10d7e1d43e614cd7026904dd8507d',  # r-inla-21.01.18.tgz
+    '0d02239274f87c2a11db55fbf2784b1a9ba8b8aa7bbf428f0e85b96e3228effe',  # r-inla-21.01.18-foss2020a-skip_binaries.patch
+]
+
+dependencies = [
+    ('R', '4.0.3'),
+    ('GSL', '2.6'),
+    ('Rmath', '4.0.3'),
+    ('jemalloc', '5.2.1'),
+    ('OpenBLAS', '0.3.12'),
+    ('SuiteSparse', '5.8.1', '-METIS-5.1.0'),
+    ('muParser', '2.2.5'),
+    ('METIS', '5.1.0'),
+    ('imkl', '2020.4.304'),
+    ('OpenSSL', '1.1.1h'),
+]
+
+# skipsteps = ['configure']
+
+files_to_copy = [(['local/%s/*' % local_files], '%s' % local_files) for local_files in ["bin", "doc", "include", "lib"]]
+
+prebuildopts = 'export INLA_TAG=Version_21.01.18 && '
+prebuildopts += "export CPATH=$EBROOTR/lib64/R/include/:$CPATH && "
+prebuildopts += "export C_INCLUDE_PATH=$PWD/local/include/:$C_INCLUDE_PATH && "
+prebuildopts += "export LD_LIBRARY_PATH=$EBROOTR/lib64/R/lib:$LD_LIBRARY_PATH && "
+prebuildopts += "rm local/lib/libamd.a local/lib/libGMRFLib.a local/bin/inla* local/lib/libmuparser.so* "
+prebuildopts += "local/lib/libtaucs.a $PWD/local/lib/pardiso-main/pardiso-main-700/libpardiso/Linux/libamd.a && "
+prebuildopts += 'mkdir -p %(installdir)s/include && '
+prebuildopts += 'mkdir -p %(installdir)s/lib && '
+prebuildopts += 'mkdir -p %(installdir)s/bin && '
+prebuildopts += 'mkdir %(builddir)s/eb_openblaso && '
+prebuildopts += 'ln -s $EBROOTOPENBLAS/lib/libopenblas.so.0 %(builddir)s/eb_openblaso/libopenblaso.so.0 && '
+prebuildopts += 'ln -s $EBROOTOPENBLAS/lib/libopenblas.so %(builddir)s/eb_openblaso/libopenblaso.so && '
+prebuildopts += "export LD_LIBRARY_PATH=%(builddir)s/eb_openblaso:$LD_LIBRARY_PATH && "
+prebuildopts += "export LD_LIBRARY_PATH=%(installdir)s/lib:$LD_LIBRARY_PATH && "
+prebuildopts += "cp ./local/lib/libpardiso.so %(installdir)s/lib && "
+
+buildopts = 'LEXTPREFIX="%(installdir)s" '
+buildopts += 'PREFIX="%(installdir)s" LEXTLIBS="%(installdir)s/lib" IEXTLIBS="%(installdir)s/include" '
+buildopts += 'MKLROOT="$MKLROOT" CMAKE_AR=gcc-ar '
```

Comment:
`$MKLROOT` is not defined when using `foss` toolchain?

Reply:
correct